weave-js/src/common/components/WBMenu.tsx

```diff
@@ -37,20 +38,26 @@ const DEFAULT_OPTION_RENDERER: OptionRenderer = ({
   selected,
   fontSize,
   lineHeight,
-}) => (
-  <S.Item
-    data-test={option['data-test']}
-    hovered={hovered}
-    fontSize={fontSize}
-    lineHeight={lineHeight}>
-    {getOptionDisplayName(option)}
-    <S.ItemIcon
-      name={
-        option.icon ?? (selected && option.icon !== null ? 'check' : 'blank')
-      }
-    />
-  </S.Item>
-);
+}) => {
+  const iconName =
+    option.icon ?? (selected && option.icon !== null ? 'checkmark' : 'blank');
+  return (
+    <S.Item
+      data-test={option['data-test']}
+      hovered={hovered}
+      fontSize={fontSize}
+      lineHeight={lineHeight}>
+      {getOptionDisplayName(option)}
+      {iconName !== 'blank' && (
+        <S.ItemIcon
+          name={iconName as IconName}
+          hovered={hovered}
+          selected={selected}
+        />
+      )}
+    </S.Item>
+  );
+};
```

Comment:
changes here are:
- use new Icon component
- only render when _not_ blank, this is because we want to avoid sentry alerts for "blank" not being a real icon. The old system would render null but not report on it.
- coalesce the iconName into `IconName`... this isn't great but a way to get passed a larger issue